packages/example-app-base/src/components/ExampleCard/ExampleCard.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { css } from 'office-ui-fabric-react/lib/Utilities';
 import './ExampleCard.scss';
-import { CommandButton } from 'office-ui-fabric-react/lib/Button';
+import { ActionButton } from 'office-ui-fabric-react/lib/Button';
```

Comment:
I agree, I could break this action button into a different PR, but it's a big breaking change, and it's needed for the command bar, so I'd need to merge action button into a 4.0 branch and then merge it right back into command bar anyway. :)